docs/components/Combobox/Web.stories.tsx

```diff
@@ -265,20 +252,17 @@ const ComboboxMultiSelection: ComponentStory<typeof Combobox> = args => {
   );
 };
 
-export const Basic = BasicCombobox.bind({});
-Basic.args = {};
-
-export const TriggerButton = ComboboxButton.bind({});
-TriggerButton.args = {};
-
-export const TriggerChip = ComboboxChip.bind({});
-TriggerChip.args = {};
-
 export const ClearSelection = ComboboxClearSelection.bind({});
 ClearSelection.args = {};
 
+export const CustomActivator = ComboboxCustomActivator.bind({});
+CustomActivator.args = {};
+
 export const EmptyState = ComboboxEmptyState.bind({});
 EmptyState.args = {};
 
 export const MultiSelect = ComboboxMultiSelection.bind({});
 MultiSelect.args = {};
+
+export const SingleSelect = ComboboxSingleSelection.bind({});
+SingleSelect.args = {};
```

Comment:
This and a few other changes are just be organizing the order of the stories and templates in this file